api/catalog/api/controllers/elasticsearch/related.py

```diff
@@ -0,0 +1,42 @@
+from elasticsearch_dsl import Search
+
+from catalog.api.controllers.elasticsearch.utils import (
+    _exclude_filtered_providers,
+    _get_query_slice,
+    _get_result_and_page_count,
+    _post_process_results,
```

Comment:
If we're importing these functions to other modules, I think it might be best to have them be public rather than "private"